<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpCommsForm</class>
 <widget class="QWidget" name="UdpCommsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="3">
    <widget class="QLabel" name="lblTxPort">
     <property name="text">
      <string>Send To</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btnPing">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Invalid IP</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../res/coretools.qrc">
       <normaloff>:/working.png</normaloff>:/working.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="spinRxPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>Port: </string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>4704</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinTxPort">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>Port :</string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>4703</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblRxPort">
     <property name="text">
      <string>Receive</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbNicList"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblLocalNIC">
     <property name="text">
      <string>Local NIC IPs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbNicList</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edTxIp">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblTxIp">
     <property name="text">
      <string>Dest IP address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>edTxIp</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="coretools.qrc"/>
 </resources>
 <connections/>
</ui>
